.github/workflows/hil_sample_zephyr.yml

```diff
@@ -105,6 +105,8 @@ jobs:
           pip3 install -r zephyr/scripts/requirements-run-test.txt
 
           pip3 install git+https://github.com/golioth/python-golioth-tools@v0.2.0
+      - name: Power Cycle USB Hub
```

Comment:
I suspect we'll also want to power cycle in between individual twister tests, but this is a good start!